src/catalog.html

```diff
@@ -24,13 +24,7 @@
               type="search"
               placeholder="Search"
             />
-            <!-- UWAGA DO PRZEKLEJENIA jasny tryb -->
-            <!-- <button class="search btn__search_black" id="searchBtn"> -->
-            <!-- <svg width="20" height="20"> -->
-            <!-- <use href="./images/icons_desktop.svg#search_icon"></use> -->
-            <!-- </svg> -->
-            <!-- </button> -->
-            <button class="search btn__search_white" id="searchBtn">
```

Comment:
czy linia <button class="search btn__search_white" id="searchBtn"> nie powinna zostać cała? zmieniłaś na <button class="search" id="searchBtn"> are you siur?

Reply:
u mnie działa na local jak powinno :)

Reply:
moglabym to zostawic i do body.light {....... dodać klasę 
ale tak tez działa, jakby co naprawie